### PR TITLE
Issue53-Clean test file

### PR DESCRIPTION
**Pull request by ArCiGoUnoSquare**
* Some constants in `Grid.spec.js` were removed and placed into files to make the file more readable
* Was added a unit test to check the top pager

### CODE REVIEW

File: test/utils/columnStructure.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+const columns = [
+  {
+    'Label': 'Order ID',
+    'Name': 'OrderID',
+    'Sortable': true,
+    'SortOrder': 1,
+    'SortDirection': 'Ascending',
+    'IsKey': true,
+    'Searchable': false,
+    'Visible': true,
```
Comment (geoperez) [inline test/utils/columnStructure.js]: You don't need to add visible true, and many other properties. Check with @jpalcala this issue.

**Comment by geoperez** [comment]
Do you reuse a PR from another branch or why I can see another history?

**Comment by ArCiGoUnoSquare** [comment]
No, @geoperez

**Comment by kadosh** [comment]
Rejected, sync with master and create a new branch to apply your changes.

**Comment by ArCiGoUnoSquare** [comment]
It was synced with master! D:

**Pull request closed without merging**